stash_plugin panics when run by hand with no arguments

Version 6.8.1 (Linux x86_64)
```
% ./stash_plugin
PANIC[2022-08-15T16:18:11-05:00] Failed to open infile: open : no such file or directory
panic: (*logrus.Entry) 0xc00011e0e0

goroutine 1 [running]:
github.com/sirupsen/logrus.(*Entry).log(0xc00011b9c0, 0x0, {0xc0000b0200, 0x37})
        /home/runner/go/pkg/mod/github.com/sirupsen/logrus@v1.8.1/entry.go:259 +0x24f
github.com/sirupsen/logrus.(*Entry).Log(0xc00011e070, 0x0, {0xc00011b9c0, 0x61b95a, 0xc0000fc040})
        /home/runner/go/pkg/mod/github.com/sirupsen/logrus@v1.8.1/entry.go:293 +0x4f
github.com/sirupsen/logrus.(*Entry).Logln(0xc00011e070, 0x0, {0xc00011bcb8, 0x0, 0x0})
        /home/runner/go/pkg/mod/github.com/sirupsen/logrus@v1.8.1/entry.go:383 +0x85
github.com/sirupsen/logrus.(*Logger).Logln(0xc0000fc000, 0x0, {0xc00011bcb8, 0x2, 0x2})
        /home/runner/go/pkg/mod/github.com/sirupsen/logrus@v1.8.1/logger.go:292 +0x65
github.com/sirupsen/logrus.(*Logger).Panicln(...)
        /home/runner/go/pkg/mod/github.com/sirupsen/logrus@v1.8.1/logger.go:333
github.com/sirupsen/logrus.Panicln(...)
        /home/runner/go/pkg/mod/github.com/sirupsen/logrus@v1.8.1/exported.go:264
main.main()
        /home/runner/work/stashcp/stashcp/cmd/stash_plugin/main.go:96 +0xb5a
```
This is probably not a thing that's commonly done but a stack trace is still kind of ugly.

--- a/cmd/stash_plugin/main.go
+++ b/cmd/stash_plugin/main.go
@@ -89,6 +89,11 @@ func main() {
 	//var downloaded int64 = 0
 	var transfers []Transfer
 
+	if len(os.Args) == 0 && (infile == "" || outfile == "") {
+		fmt.Fprint(os.Stderr, "No source or destination specified\n")
+		os.Exit(1)
+	}
+
 	if len(os.Args) == 0 {
 		// Open the input and output files
 		infileFile, err := os.Open(infile)

